internal/provider/resource_deployment.go

```diff
@@ -107,15 +111,15 @@ func resourceDeploymentCreate(ctx context.Context, d *schema.ResourceData, meta
 		return diag.FromErr(err)
 	}
 
-	// TODO: id generation - is there a natural ID we can use?
-	d.SetId(fmt.Sprintf("%s/%s", modelUUID, deployedName))
+	id := fmt.Sprintf("%s:%s", modelName, appName)
```

Comment:
Is this intentional, to replace modelUUID with model Name?

The read function will need to add a step to resolve the UUID if this is to be the new standard